### ISSUE TITLE
library return null object reference error

### ISSUE THREAD

**Issue by pishguy** [stefanhaustein/HtmlView2#20]
for this html code:

`text = "<p style='font-size:15px;text-align:right'><br/><h2 style='text-align:right'>hello world</h2><br/></p>";
`
or

`text = "<p style='font-size:15px;text-align:right'><br></br><h2 style='text-align:right'>hello world</h2><br></br></p>";`

i get this error:

`Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.kobjects.css.CssStyleDeclaration.isSet(org.kobjects.css.CssProperty)' on a null object reference
`

my code:

```
    private void showContentPreviewAsHtml() {
        try {
            text = "<p style='font-size:15px;text-align:right'><br></br><h2 style='text-align:right'>hello world</h2><br></br></p>";
            Log.e("text ", text);
            Reader reader = new StringReader(text);

            HtmlView htmlView = new HtmlView(context);
            htmlProcessor.parse(reader, htmlView);

            ScrollView scrollView = new ScrollView(context);
            scrollView.addView(htmlView);
            markdownViewContent.addView(scrollView);

        } catch (Exception e) {
            e.printStackTrace();
            Log.e("HtmlView ", e.getMessage());
            throw new RuntimeException(e);
        }
    }
```

problem is break line `<br/>`, `<br></br>`, how can i use break line on start or end of html tags?

**Comment by stefanhaustein** [comment]
Works for me at head (see attached image). Are you sure you are using the latest version?
<img width="1056" alt="issue20" src="https://cloud.githubusercontent.com/assets/4282319/26560230/2397dcbc-44b5-11e7-9281-3d8275c2daf5.png">

**Comment by MahdiPishguy** [comment]
@stefanhaustein whats your checked version? i get error again, i installed library from gradle

**Comment by pishguy** [comment]
@stefanhaustein problem is in you library on `Gradle` sir, i download library source code and check again, i dont have any problem now, please update `gradle` repository

**Comment by stefanhaustein** [comment]
Ooops, thanks for the hint, released v2.0.2

**Comment by pishguy** [comment]
@stefanhaustein thanks
